### PR TITLE
Wiki Home Page Update

### PR DESCRIPTION
**Pull request by Naterater**
Some organizational clarifications and updates to help new users understand what Ardupilot is.

### CODE REVIEW

**Comment by auturgy** [comment]
Thanks for doing this! As it’s a main entry point for users, a couple of us need to have a look before merging, but it’s definitely an improvement.

Regards,

James

File: ardupilot/source/index.rst [hunk 1]
```diff
@@ -20,11 +19,15 @@
     <td width="27%">
 
 
-:ref:`Hardware <stores>` - The embedded systems and
-peripheral sensors that act as the vehicle's brain, eyes, ears, etc.
+:ref:`Hardware <stores>`  is the peripheral sensors, conrtoller, and output
```
Comment (khancyr) [inline ardupilot/source/index.rst]: controller

File: ardupilot/source/index.rst [hunk 2]
```diff
@@ -57,15 +59,16 @@ update is all it takes to repurpose your hardware into a different role.
...
 
-Initial set-up, configuration, and testing. Mission-planning/operation,
+A GCS allows users to set-up, configure, test, and tune the vehicle. Advanced
+packages allow autonomous mission planning, operation,
 and post-mission analysis.
 
-Point-and-click intuitive interaction with your hardware, or advanced
-custom scripting for niche mission profiles. Options are everything with
-ArduPilot.
+:ref:`Mission Planner <planner:home>` is the full-featured GCS supported by Ardupilot. It offers
```
Comment (khancyr) [inline ardupilot/source/index.rst]: a full

File: ardupilot/source/index.rst [hunk 3]
```diff
@@ -57,15 +59,16 @@ update is all it takes to repurpose your hardware into a different role.
     </td>
     <td width="27%">
 
-:ref:`Software <planner:home>`- Your interface to the
-hardware.
+:ref:`Software <http://ardupilot.org/copter/docs/common-choosing-a-ground-station.html>` is your interface to the
```
Comment (khancyr) [inline ardupilot/source/index.rst]: should be sphinx href

File: ardupilot/source/index.rst [hunk 4]
```diff
@@ -83,8 +86,11 @@ ArduPilot.
 
 Community:
 ==========
+.. tip::
 
-Community is what really sets ArduPilot apart from the closed-source offerings in the market. An area for open discourse on the ArduPilot projects direction between developers and its customers.  Here are some of the places you can find ardupilot users and developers:
+   Keep up with the latest ArduPilot related blogs on `ArduPilot.org! <http://ardupilot.org/>`__
```
Comment (khancyr) [inline ardupilot/source/index.rst]: change link to blog

**Comment by magicrub** [comment]
@khancyr I made those changes

File: ardupilot/source/index.rst (hunk 3 above)
Comment (magicrub) [reply ardupilot/source/index.rst]: fixed

File: ardupilot/source/index.rst (hunk 4 above)
Comment (magicrub) [reply ardupilot/source/index.rst]: fixed

File: ardupilot/source/index.rst (hunk 2 above)
Comment (magicrub) [reply ardupilot/source/index.rst]: fixed

File: ardupilot/source/index.rst (hunk 1 above)
Comment (magicrub) [reply ardupilot/source/index.rst]: fixed

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**